Sources/ViewInspector/SwiftUI/CustomView.swift

```diff
@@ -126,21 +128,6 @@ public extension NSViewControllerRepresentable where Self: Inspectable {
     }
 }
 
-@available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
```

Comment:
These conformances are no longer necessary since we threw these errors in `ContentExtractor`.

https://github.com/nalexn/ViewInspector/blob/47d11e623ddd0477115df60aaa81fdd9c953c9cf/Sources/ViewInspector/ContentExtraction.swift#L41-L70